indico/modules/events/layout/forms.py

```diff
@@ -135,10 +135,10 @@ class CSSForm(IndicoForm):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.css_file.description = _('If you want to fully customize your conference page you can create your own '
-                                      'stylesheet and upload it. An example stylesheet can be downloaded '
-                                      "<a href='{base_url}/standard.css' target='_blank'>here</a>."
-                                      .format(base_url=config.CONFERENCE_CSS_TEMPLATES_BASE_URL))
+        self.css_file.description = (_('If you want to fully customize your conference page you can create your own '
+                                       'stylesheet and upload it. An example stylesheet can be downloaded '
+                                       "<a href='{base_url}/standard.css' target='_blank'>here</a>.")
```

Comment:
let's maybe move the whole link outside the i18n string, ie similar to what we did here?

```
        message = _('Please note that even though your request has been accepted, you still have to '
                    'request badges for each one of your participants. {link}More details here.{endlink}').format(
                        link=f'<a href="{link}">',
                        endlink='</a>')
```